src/Lumi/Components/Modal.purs

```diff
@@ -229,66 +234,83 @@ modalPortal = toReactComponent identity modalPortalComponent
 
     closeModal = runEffectFn2 toggleBodyClass "react-modal-open" false
 
-    render { props, state } =
+    render self@{ props, state } =
```

Comment:
You don't appear to be using `self` so I think the change to this line can be reverted?